src/custom/api/gnosisProtocol/api.ts

```diff
@@ -290,13 +304,13 @@ async function _handleQuoteResponse<T = any, P extends QuoteQuery = QuoteQuery>(
 
       const sentryError = new Error()
       Object.assign(sentryError, quoteError, {
-        message: `Error querying fee from API - sellToken: ${sellToken}, buyToken: ${buyToken}`,
-        name: 'FeeErrorObject',
+        message: `Error querying fee from API - sellToken: ${sellToken}, buyToken: ${buyToken}, status code: ${response.status}`,
+        name: 'HandleQuoteResponse',
       })
 
       // report to sentry
       Sentry.captureException(sentryError, {
-        tags: { errorType: 'getFeeQuote' },
+        tags: { errorType: 'handleQuoteResponse', backendErrorCode: response.status },
```

Comment:
I see u said in the description `handleQuoteResponse`  is a better name. Is it because u prefer to call it like the auxiliary function? I'm not 100% is better, i kind of liked that is called after the function that is related to the error handling

Reply:
you're right - changed name. uses now the quote error type (should it exist)